src/applications/gi-sandbox/components/LearnMoreLabel.jsx

```diff
@@ -34,12 +33,7 @@ export default function LearnMoreLabel({
           'vads-u-font-weight--bold': bold,
         },
       )}
-      onClick={() => {
-        focusElement(labelFor);
-        recordEvent({
-          event: `Learn more clicked for ${labelFor}`,
```

Comment:
Why is this being deleted?

Reply:
That was being removed because it was showing up across all of the comparison tool. I moved that event to the profile page specifically.